### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by mhaas**

### CODE REVIEW

File: docs/source/release_process.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Creating a new Release of the SDK
```
Comment (karthikmswamy) [inline docs/source/release_process.rst:1]: Just a minor point: would this be more clear about which SDK - `Creating a new release of the Data Attribute Recommendation Python SDK`?

Comment (karthikmswamy) [reply docs/source/release_process.rst:1]: Also, I do not feel release needs to be capitalised, but this is just my opinion.

**Comment by karthikmswamy** [comment]
By the way, this is really well documented and very useful / important information. Thanks for adding this :)

**Comment by mhaas** [comment]
Hi @karthikmswamy, thanks for the pointers. Does the latest update clarify your questions?

**Comment by mhaas** [comment]
Fixes #22

**Pull request merged**